tests/datasets/test_members_view.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime
+
+import pytest
+from django.urls import reverse
+from django_webtest import DjangoTestApp
+
+from vitrina.datasets.factories import DatasetFactory, DatasetMemberFactory
+from vitrina.datasets.models import DatasetMember
+from vitrina.orgs.factories import OrganizationFactory, RepresentativeFactory
+from vitrina.users.models import User
+
+
+@pytest.mark.django_db
+def test_dataset_members_view_no_login(app: DjangoTestApp):
+    org = OrganizationFactory()
+    dataset = DatasetFactory(organization=org)
```

Comment:
```suggestion
    dataset = DatasetFactory()
```



Organization should be created automatically by factory boy

https://factoryboy.readthedocs.io/en/stable/reference.html#subfactory